keyboards/ogurec/left_pm/readme.md

```diff
@@ -0,0 +1,13 @@
+# ogurec keymap for left-sided PM placement
+
+Make example for this keyboard (after setting up your build environment):
+
+    make ogurec:default
```

Comment:
```suggestion
    make ogurec/left_pm:default
```

Reply:
I actually specify DEFAULT_FOLDER = ogurec/left_pm in rules.mk. Should I commit this suggested change and remove this form rules.mk?

Reply:
No, it's just that you have three readmes now, so the left and right specific ones should have the correct make command (even if left is default).